plugins/in_tail/tail.c

```diff
@@ -613,6 +613,14 @@ static struct flb_config_map config_map[] = {
     },
 #endif
 
+#ifdef FLB_HAVE_UTF8_ENCODER
+    {
+     FLB_CONFIG_MAP_STR, "encoding", NULL,
+     0, FLB_FALSE, 0,
+     "specify the charset encoder to decode message",
```

Comment:
Perhaps to clarify the wording for the encoding parameter:
```specify the input encoding for converting to UTF-8```

Reply:
Changed text tail.c test to `specify the input encoding for converting to UTF-8`